pages/admin/request/create-renewal.tsx

```diff
@@ -473,7 +476,7 @@ export default function CreateRenewal() {
                       height="48px"
                       width="180px"
                       type="submit"
-                      loading={loading}
+                      isLoading={submitRequestLoading}
```

Comment:
Let's also use `submitRequestLoading` to disable the "Back to search" and "Discard request" buttons when the mutation is loading